charts/dapr/README.md

```diff
@@ -126,6 +126,7 @@ The Helm chart has the follow configuration options that can be supplied:
 | `dapr_operator.runAsNonRoot`              | Boolean value for `securityContext.runAsNonRoot`. You may have to set this to `false` when running in Minikube | `true` |
 | `dapr_operator.resources`                 | Value of `resources` attribute. Can be used to set memory/cpu resources/limits. See the section "Resource configuration" above. Defaults to empty | `{}` |
 | `dapr_operator.debug.enabled`             | Boolean value for enabling debug mode | `{}` |
+| `dapr_operator.serviceReconciler`         | Boolean value for enabling the reconciler that creates Services for Dapr-enabled Deployments and StatefulSets.<br>Note: disabling this could prevent Dapr service invocation from working. | `true` |
```

Comment:
should this key be `dapr_operator.serviceReconciler.enabled` ?
`dapr_operator.runAsNonRoot` implicitly means that it is a bool value. But the above does not.
Having enabled or disabled would be better here IMO...

Reply:
Done (using `disabled` as per below)